scripts/setup.sh

```diff
@@ -15,13 +15,17 @@
 # limitations under the License.
```

Comment:
Could we add a new Makefile target for that? 

.PHONY: test-e2e-local
test-e2e-local: ## It will run the script to install kind and run e2e tests         
	./test_e2e_local.sh

Reply:
Also could you add new makefile target in : https://github.com/kubernetes-sigs/kubebuilder/blob/master/CONTRIBUTING.md#what-to-do-before-submitting-a-pull-request